### PR TITLE
Details car

### PR DESCRIPTION
**Pull request by maksymvasylyev**

### CODE REVIEW

File: components/FilterForm/FilterForm.js [hunk 1]
```diff
@@ -61,9 +61,14 @@ function FilterForm() {
           between(car.Price, Number(minPrice), Number(maxPrice))
       )
     );
-    event.target.reset();
+    // event.target.reset();
```
Comment (onemanwenttomow) [inline components/FilterForm/FilterForm.js:64]: If this is commented out, do we need it anymore?

File: components/FilterForm/FilterForm.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ function between(x, min, max) {
 }
 
 function FilterForm() {
-  const [list, setList] = useState(data);
+  const [list, setList] = useLocalStorageState(data, { defaultValue: [] });
```
Comment (onemanwenttomow) [inline components/FilterForm/FilterForm.js:44]: I think the first argument to useLocalStorageState needs to be the `key` and not the data. The second argument needs to be where we would put the data. i.e.

``` js
  const [list, setList] = useLocalStorageState("list", { defaultValue: data });
```

File: components/FilterForm/FilteredCarsList.js [hunk 3]
```diff
@@ -19,13 +20,15 @@ function FilteredCarsList({ list }) {
...
-              <Image
-                style={{ borderRadius: "10%" }}
-                src={car.imageSource}
-                alt={car.model}
-                width={300}
-                height={150}
-              />
+              <Link href={`/detailsCar/${car.id}`}>
+                <Image
+                  style={{ borderRadius: "10%" }}
```
Comment (onemanwenttomow) [inline components/FilterForm/FilteredCarsList.js:25]: There seems to be a mix of styled components and inline styles in your project. Is there a reason for this?

If you want to use styled components on the Image component from nextjs i think you can do something like this. 

``` js
const StyledImage = styled(Image)`
  border-radius: 10%;
`;
```

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Nice job! I added some comments

File: components/FilterForm/FilteredCarsList.js (hunk 3 above)
Comment (maksymvasylyev) [reply components/FilterForm/FilteredCarsList.js:25]: Thank you very much!
I really appreciate your work by reviewing my PR. I have fixed comments and will work more on styling after functionality is ready. Thanks again!

**Pull request merged**